crates/notion-core/src/version/mod.rs

```diff
@@ -74,7 +74,7 @@ fn version_req_parse_error(error: &ReqParseError) -> ErrorDetails {
     }
 }
 
-fn version_parse_error(error: &SemVerError) -> ErrorDetails {
+pub fn version_parse_error(error: &SemVerError) -> ErrorDetails {
```

Comment:
Another spot where I don't think we need `pub`

Reply:
This is also used in `crates/notion-core/src/inventory/serial.rs`, so this needs `pub` - I'll change to `pub(crate)`